src/scripts/docs/IconApp.jsx

```diff
@@ -17,9 +17,9 @@ const IconScope = {
 
 // Add new icons here
 const icons = [
-  'analytics',
   'add-circle',
   'alert-triangle',
+  'analytics',
```

Comment:
Sorted icons alphabetically for easier readablity, as well as added the two new ones